devices.json

```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "id": "Nexus 4",
+    "adb": {
+      "ro.product.device": [ "mako" ],
+      "ro.bootloader": "MAKOZ20i",
+      "ro.build.id": "JSS15J"
+    },
+    "fastboot": {
+      "product": [ "mako" ]
```

Comment:
No way we can check the bootloader version ? I'm pretty sure KK and L might differ.